pkg/kotsadmsnapshot/backup.go

```diff
@@ -962,6 +958,12 @@ func prepareIncludedNamespaces(namespaces []string) []string {
 		uniqueNamespaces[n] = true
 	}
 
+	if isEC {
+		uniqueNamespaces["embedded-cluster"] = true
+		uniqueNamespaces["kube-system"] = true
+		uniqueNamespaces["openebs"] = true
```

Comment:
wouldn't `openebs` be installed along with k0s and velero in order to even initiate the restore process?

Reply:
I think we install k0s + velero, then restore openebs, then restore kotsadm

Reply:
how would velero be installed without openebs?

Reply:
velero does not require in-cluster storage

Reply:
👍